model.go

```diff
@@ -89,9 +89,27 @@ type Game struct {
 	RollCount int
 }
 
+// Feature represents the features available for the game.
+type Feature string
+
+// Available features
+const (
+	SixDice Feature = "six-dice"
+)
+
+func Features() []Feature {
+	return []Feature{
+		SixDice,
+	}
+}
+
 // NewGame initializes an empty Game.
-func NewGame() *Game {
-	dd := make([]*Dice, NumberOfDices)
+func NewGame(features []Feature) *Game {
```

Comment:
instead of changing `NewGame()` it would be better to have a separate `NewGameWithFeature(ff ...Feature) *Game`